::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Thu, 23 Feb 2017 23:45:41 +0000 (+0100)
Subject: install_quagga.sh: Enable parallel build

install_quagga.sh: Enable parallel build

Change-Id: I8b961ebacc6ab096200d1e124d8f50ed6e2b8217
Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---

diff --git a/pre_build_hook b/pre_build_hook
index ab8b0a6..20dd715 100755
--- a/pre_build_hook
+++ b/pre_build_hook
@@ -78,3 +78,5 @@ sed -i 's/cd zrpcd/cd zrpcd \&\& git checkout eddf5bd6305ef52a2d39df99860c734935
 sed -i 's/quagga_110_mpbgp_capnp/95bb0f4a5/' deployment_scripts/install_quagga.sh
 # pin thrift to the latest version at this point in time
 sed -i 's/cd thrift/cd thrift \&\& git checkout 0.10.0/' deployment_scripts/install_quagga.sh
+# enable parallel build
+sed -i 's|make$|make -j|g' deployment_scripts/install_quagga.sh
